src/pages/home/report/ReportActionsView.js

```diff
@@ -112,85 +83,11 @@ function ReportActionsView(props) {
         Report.openReport(reportID);
     };
 
-    useEffect(() => {
-        const unsubscribeVisibilityListener = Visibility.onVisibilityChange(() => {
-            if (!isReportFullyVisible) {
-                return;
-            }
-            // If the app user becomes active and they have no unread actions we clear the new marker to sync their device
-            // e.g. they could have read these messages on another device and only just become active here
-            const hasUnreadActions = ReportUtils.isUnread(props.report);
-            if (!hasUnreadActions) {
-                setNewMarkerReportActionID('');
-            }
-        });
-        return () => {
-            if (!unsubscribeVisibilityListener) {
-                return;
-            }
-            unsubscribeVisibilityListener();
-        };
-    }, [isReportFullyVisible, isFocused, props.report, setNewMarkerReportActionID]);
-
     useEffect(() => {
         openReportIfNecessary();
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
-    useEffect(() => {
-        // Why are we doing this, when in the cleanup of the useEffect we are already calling the unsubscribe function?
-        // Answer: On web, when navigating to another report screen, the previous report screen doesn't get unmounted,
-        //         meaning that the cleanup might not get called. When we then open a report we had open already previosuly, a new
-        //         ReportScreen will get created. Thus, we have to cancel the earlier subscription of the previous screen,
-        //         because the two subscriptions could conflict!
-        //         In case we return to the previous screen (e.g. by web back navigation) the useEffect for that screen would
-        //         fire again, as the focus has changed and will set up the subscription correctly again.
-        const previousSubUnsubscribe = newActionUnsubscribeMap[reportID];
-        if (previousSubUnsubscribe) {
-            previousSubUnsubscribe();
-        }
-
-        // This callback is triggered when a new action arrives via Pusher and the event is emitted from Report.js. This allows us to maintain
-        // a single source of truth for the "new action" event instead of trying to derive that a new action has appeared from looking at props.
-        const unsubscribe = Report.subscribeToNewActionEvent(reportID, (isFromCurrentUser, newActionID) => {
```

Comment:
Hi, coming from https://github.com/Expensify/App/issues/26432

The issue is a regression from this removed, where the current user doesn't scroll to the bottom after sending a message.

I have several questions regarding these changes:
1. Are the above results expected?
2. If not, can we reintegrate the listener `Report.subscribeToNewActionEvent` to scroll the current user when sending a message? Or there's a performance consideration to use that listener.

cc @gedu @MonilBhavsar @allroundexperts

Reply:
> Are the above results expected?

Sorry, what result?

Reply:
Ahh, sorry for not making it clear.

> Sorry, what result?

The current user doesn't scroll to the bottom after sending a message when they're not at the bottom of the chat.

Reply:
That does seem like issue. We should definitely scroll user to the bottom

Reply:
@MonilBhavsar Is there any drawback or performance issue if we use `Report.subscribeToNewActionEvent` to get the current report action?

Reply:
I don't think it will have an impact on performance, it will be mainly just to know when a new message is sent. Because this newActionEvent only listens to comments sent, and not received from others.